spec/services/patient_deduplication/deduplicator_spec.rb

```diff
@@ -16,6 +16,11 @@ def with_comparable_attributes(related_entities)
   end
 end
 
+def duplicate_records(deduped_record_ids)
```

Comment:
I'm not sure if it would have any value outside of the specs, but this could also be a class method on DeduplicationLog.